source/preferences/manage-your-desktop-notifications.rst

```diff
@@ -0,0 +1,91 @@
+Manage your desktop notifications
+=================================
+
+.. include:: ../_static/badges/allplans-cloud-selfhosted.rst
+  :start-after: :nosearch:
+
+.. |dot-badge| image:: ../images/dot-badge.png
+  :alt: A dot on the badge means you have unread activity in at least one channel you're a member of.
+  :width: 50px
+
+.. |numbered-badge| image:: ../images/numbered-badge.png
+  :alt: A numbered badge means you have at least 1 unread message, @mention, or one of your keywords has triggered a notification.
+  :width: 50px
+
+Enable notifications
+--------------------
+
+From Mattermost v9.9 and desktop app v5.5, Mattermost prompts you to enable notifications in the desktop app the first time you connect to a Mattermost server.
+
+- When you select **Allow**, you won't be asked again. You'll start receiving notifications in the desktop app for all Mattermost activity with `badges <#badge-based-notifications>`__, `banner alerts <#banner-alerts>`__ and `sounds <#notification-sounds>`__. See the section below on `customizing your notifications <#customize-your-notifications>`__ based on how you prefer to be notified about Mattermost activity in the desktop app.
+- If you select **Deny** or **Deny Permanently**, you won't be asked again. You won't receive Mattermost notifications in the web browser. You can change this preference by :ref:`editing the server connection <preferences/connect-multiple-workspaces:edit a server>` to :ref:`manage your notification permissions <preferences/connect-multiple-workspaces:manage system permissions>`.
+
+.. note::
+
+  - You may also need to enable notifications in Windows, macOS, or Linux for Mattermost by changing your System Preferences.
+  - When you allow desktop notifications, you'll also receive a second prompt asking you to allow notifications in your default web browser.
+
+    .. image:: ../images/web-notification-prompt.png
+      :alt: An example of a Firefox web browser notification prompt.
```

Comment:
This screenshot is for web browsers that doesn't apply for desktop. For desktop we would get a dialog like: https://github.com/mattermost/docs/pull/7378#discussion_r1765691692